### ISSUE TITLE
allow inherit in copy?

### ISSUE THREAD

**Issue by sedlund** [creativeprojects/resticprofile#147]
looking to do something like this:

```yaml
password:
  password-file: pass.txt

local:
  inherit: password
  repository: ~/backups/restic

remote:
  inherit: password
  repository: s3:https://minio.myserver.com/restic
  env:
    AWS_ACCESS_KEY_ID: 000
    AWS_SECRET_ACCESS_KEY: FFF

default:
  inherit: local
  backup:
    source: ~/
    exclude-file: excludes.txt
  copy:
    inherit: remote
```

or is there a better way to do this?

Thanks for the cool project!

**Comment by sedlund** [comment]
I tried to do this with templates, but of course this makes the file not valid yaml any longer which makes the LSP blowup.  Then finding that the indentation of the template definition matters when it is rendered in different scopes made it a no go.

Then I found the new [mixins](https://creativeprojects.github.io/resticprofile/configuration/inheritance/#mixins)and it was the answer to all the worlds problems!

Thank you so much.
